src/app/components/annotations/pages/search/search.component.html

```diff
@@ -18,25 +18,43 @@ <h1>Annotation Search</h1>
       Download Annotations
     </a> -->
 
-    <button class="btn btn-primary" (click)="navigateToVerificationGrid()">
-      Verify This List
-    </button>
+    <!--
+      We add the tooltip to a wrapping span because if the button is disabled
+      ng-boostrap will not show the tooltip
+     -->
+    <span
+      [ngbTooltip]="
+        session.isLoggedIn
+          ? 'Verify the annotations in the list'
+          : 'You must be logged in to verify annotations'
+      "
```

Comment:
this seems like it should be a reusable component (disabled unless logged in and tooltip to explain it)